ms2/src/org/labkey/ms2/loadGoAutomatic.jsp

```diff
@@ -43,8 +45,8 @@ LabKey database. If you click "Continue", this server will automatically:
     <li><%=h(loaded ? "Reload" : "Load")%> the database tables with information from the downloaded GO files</li>
 </ul>
 
-Your LabKey Server must be able to connect via the Internet to ftp.geneontology.org. Assuming a reasonably fast
-Internet connection, this process should take less than five minutes to complete. The loading will take place
+Your LabKey Server must be able to connect via the Internet to geneontology.org. This process should take less than
+    five minutes to complete. The loading will take place
 in the background and you can continue to use your LabKey Server normally. If you want, you can monitor the
 process by refreshing the status information on the next page.<br><br>
 

```

Comment:
Should remove the Windows-only FTP warning section below

Reply:
Done.